tests/test_algorithms.py

```diff
@@ -83,3 +83,51 @@ def test__split_tests_calculates_avg_test_duration_only_on_present_tests(self, a
         expected_first, expected_second = expected
         assert first.selected == expected_first
         assert second.selected == expected_second
+
+    def test_each_group_is_assigned_a_test_front_loaded(self):
+        from collections import namedtuple
+
+        from pytest_split import algorithms
```

Comment:
Is there some motivation behind inline imports?

Reply:
moved 👍